### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by Felipeg005**
## In order to achieve project requirements next actions were taken:

- [x] Set up React Testing Library Following the instructions in the official documentation.
- [x] Write unit tests for the files operate.js and calculate.js using Jest.
- [x] Create unit tests for all React components:
- [x] Use Jest snapshots to test the components.
- [x] Run your full tests suite using Jest. All tests should pass.

[Here's the link to the live version.](https://felipeg005.github.io/Math-Magicians-React-App/)

Thanks for your review! 😄

### CODE REVIEW

File: src/__test__/components.test.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+describe('Test snapshot', () => {
+  test('Render Home', () => {
+    const component = render(<Home />);
+    expect(component.container).toHaveTextContent('Welcome to our page!');
+  });
+
+  test('Render Calculator', () => {
+    const component = render(<Calculator />);
+    expect(component.container).toHaveTextContent(`Let's do some Math!`); // eslint-disable-line
```
Comment (eliasecasta) [inline src/__test__/components.test.js]: - [x] Please don't overuse the eslint disable, what issue is eslint giving you guys in here? Try to solve it instead please to comply with Microverse requirements.

File: src/__test__/calculate.test.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
+import calculate from '../logic/calculate';
+
```
Comment (eliasecasta) [inline src/__test__/calculate.test.js:2]: - [x] Please add tests for all the functionalities of your calculate script, this includes +/-, division, and division by 0 operations to comply with the project requirements: "All the functionalities & scenarios of the operate.js and calculate.js modules should be covered using Jest."

File: src/__test__/operate.test.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import operate from '../logic/operate';
+
```
Comment (eliasecasta) [inline src/__test__/operate.test.js:2]: - [x] Please also cover the operation % to comply with the project requirements.

**Review by eliasecasta — CHANGES_REQUESTED** [review]
# Status:  Changes Required ♻️
_Check the comments under the review_
<hr>

Hello @JuliCarracedo and @Felipeg005, I am Elias, your reviewer for this PR

Well done so far with your project!

I can see that you have put a great effort so far in here, the tests provided are working as expected, great job!

I will mention to you a small summary of things that need to be changed in order to keep moving:

- [x] Please remove the overuse of eslint disable to comply with microverse requirements
- [x] Don't forget about negative tests, they are required for this project, please add them to comply with the requirements.
- [x] Please check the comments below for more details.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please correct those issues and send another review request, good luck, and well done so far! 

Code review by Elias Castañeda

_(Feel free to leave any questions or comments in the PR thread if something is not 100% clear)_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561) you have a limited number of code reviews per project (you can check the exact number in your Dashboard). If you think that the code review was not fair, you can [request a second opinion using this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/__test__/components.test.js (hunk 1 above)
Comment (Felipeg005) [reply src/__test__/components.test.js]: You're totally right, the changes are done 😄

File: src/__test__/calculate.test.js (hunk 2 above)
Comment (Felipeg005) [reply src/__test__/calculate.test.js:2]: We added the required test to the calculate script, thank you.

File: src/__test__/operate.test.js (hunk 3 above)
Comment (Felipeg005) [reply src/__test__/operate.test.js:2]: We are now covering the % operation in the project, thanks for the advice.

**Review by Emmyn5600 — APPROVED** [review]
Hi,

Good job so far! 👏🏼 @e71az and @JuliCarracedo 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- All tests are passing successfully:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous reviewer.

 #### Optional suggestions
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**